pkg/api/app_peers.go

```diff
@@ -99,16 +104,16 @@ type PeersSuspendedResponse struct {
 	Peers []string `json:"peers"`
 }
 
-func (a *App) PeersSuspended() (*PeersSuspendedResponse, error) {
+func (a *App) PeersSuspended() (PeersSuspendedResponse, error) {
```

Comment:
the same about errors

Reply:
are you sure there should be a particular function for calling `a.peers.Suspended()`?